src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -486,8 +486,7 @@ ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v3::Bucketize>& n
     InferenceEngine::Blob::Ptr blobPtr;
     switch (port) {
         case 0: {
-            unsigned long data_size = std::accumulate(begin(targetShape), end(targetShape), 1, std::multiplies<uint64_t>());
-            return create_and_fill_tensor(elemType, ov::Shape{data_size * 5}, 0, 10, 7235346);
+            return create_and_fill_tensor_unique_sequence(elemType, targetShape, 0, 10, 7235346);
```

Comment:
How many dimensions does the targetShape have here for port 0 and 1 ?

Reply:
In both existing tests only one port (only port 0), its size 1*1